nannou/src/geom/path.rs

```diff
@@ -95,6 +95,14 @@ impl Builder {
         lyon::path::builder::Flattened::new(self, tolerance)
     }
 
+    /// Sets the position in preparation for the next sub-path.
+    ///
+    /// If the current sub-path contains edges, this ends the sub-path without closing it.
+    pub fn begin(mut self, to: Point2) -> Self {
```

Comment:
We could also keep the `move_to` name here, to avoid breaking API. Or even having both `move_to` and `begin` and deprecating the former?

Reply:
I'm open to re-adding `move_to` with a deprecation notice if you'd like, but otherwise I'm happy for us to keep a minimal set of methods that follow what lyon itself exposes.